catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -160,7 +182,9 @@ private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable>
                 operationPolicy.putAll(policy);
             }
         }
-        return new PolicyResponseImpl(operationPolicy, operationPolicy);
+        itemPolicy.putAll(operationPolicy);
```

Comment:
I think it is redundant and was probably redundant before as well. If the user or the system cannot imply the operation policy, then the entire operation is dropped on the floor. Adding that same policy to the items would have no effect because you wouldn't even get to the point where you were considering them individually.